server/server.go

```diff
@@ -21,14 +21,21 @@ import (
 	sqle "github.com/dolthub/go-mysql-server"
 )
 
+type ServerEventListener interface {
+	ClientConnected()
```

Comment:
Seems like you need connection identifiers here to be able to give info on e.g. current connections, connection duration, etc